modules/islandora_audio/js/audio.js

```diff
@@ -4,44 +4,87 @@
  * @file
  * Displays Audio viewer.
  */
-(function ($, Drupal) {
-    'use strict';
+(function (Drupal, once) {
+  Drupal.behaviors.customBehavior = {
```

Comment:
Should probably be module namespaced in some way, instead of just `customBehavior`?

```suggestion
  Drupal.behaviors.islandora_audio_viewer = {
```